HACKING.pod

```diff
@@ -22,13 +22,39 @@ here. However, all new code should be made to comply.
 
 =head1 GETTING STARTED
 
-In the B<dev_scripts> directory you will find scripts that are meant to be
-useful for developing. Most of them are tools for creating and using
-Munin in a sandbox.
+
+=head2 GITHUB
+
+The main development of Munin is happening on L<https://github.com>. With a
+free account you can report feature requests and issues
+(L<https://github.com/munin-monitoring/munin/issues/new/choose>) and create
+pull requests (L<https://github.com/munin-monitoring/munin/pulls>).
+
+There are two interesting branches:
+
+=over
+
+=item I<stable-2.0>
+
+As the name suggests this is the stable branch for versions 2.0.x.
+Please base your plugin and bug-fix patches on this branch.
```

Comment:
Indeed I usually advertised this policy (plugin improvements go to `stable-2.0`). But @steveschnepp and me just realized, that we have diverging opinions on that matter. I hope, we will find the time to discuss this tomorrow in the weekly meeting. Thus the final text of this paragraph will need to be decided in IRC and not in this pull request, I guess. (just my opinion - we will see)

I always had a bad feeling about telling people to do some rebase without being able to point to a written documentation detailing this. Thank you for reducing this pain in the future!

Reply:
Update of our discussion: during the [IRC meeting on the 5th of September](http://meetbot.debian.net/munin/2018/munin.2018-09-05-18.29.log.html) we agreed on only applying fixes (both for core and for plugins) to `stable-2.0` from now own. These changes will be merged into `master` from time to time. All other changes go directly to `master`.